tests/php/Page/SiteTreeTest_NamespaceMapTestNode.php

```diff
@@ -5,7 +5,7 @@
 use SilverStripe\CMS\Model\SiteTree;
 use SilverStripe\Dev\TestOnly;
 
-class SiteTreeTest_NamespaceMapTest extends SiteTree implements TestOnly
+class SiteTreeTest_NamespaceMapTestNode extends SiteTree implements TestOnly
```

Comment:
Because this file ends in "Test" - phpunit though it was a test class and complained that it had no test cases. So I simply renamed it